sdk/webpubsub/azure-messaging-webpubsub/src/test/java/com/azure/messaging/webpubsub/WebPubSubServiceClientTests.java

```diff
@@ -334,9 +333,11 @@ public void testAadCredential() {
     }
 
     @Test
-    @DoNotRecord
+    @Disabled("This test requires a connectionId with SEND_TO_GROUP permission.")
```

Comment:
Same here - disable this only in live mode. We should still run this in playback mode.